weblab/config/settings/test.py

```diff
@@ -1,4 +1,15 @@
-from .base import *  # noqa
+from .base import TEMPLATES  # noqa
```

Comment:
You need the `*` here or most of the test settings won't be defined.

I don't know why the `RuntimeError` is happening though! Do you get any errors running `pytest` locally? Travis is just running `pytest --cov=weblab --cov-config=weblab/.coveragerc weblab`

Reply:
Hmm, maybe we need to add
```
- export DJANGO_SETTINGS_MODULE="config.settings.test"
```
to the script section of .travis.yml ?

Reply:
(looking at https://pytest-django.readthedocs.io/en/latest/configuring_django.html)

Reply:
Ooh, and skimming the docs I've found https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest.mark.ignore_template_errors. So in theory we can avoid touching `test.py` and instead change .travis.yml to call
```
pytest --fail-on-template-vars --cov=weblab --cov-config=weblab/.coveragerc weblab
```

Reply:
great thanks